### PR TITLE
fix: emit hot-reload line-map hint on pause-point resolve failure

### PR DESCRIPTION
**Pull request by hatayama**
## Summary
- When a pause point cannot resolve `--line` on a file that already has active hot-reload patches, the failure response now includes the compiled-line-map warning.
- Agents can see that `--line` is still mapped against the last compile, instead of getting an empty Warning.

## User Impact
- Before: resolve failure returned `Warning` empty even when the file had active hot-reload patches, so the same line-map hint already shown on success was missing on failure.
- After: that failure response carries the same compiled-line-map warning as a successful enable on an unpatched method.

## Changes
- Share `BuildCompiledLineMapWarningOrEmpty` between the enable success path and the resolve-failure path.
- Add unit tests for the helper: formatted warning, backslash normalization, and empty when no patches are active.

## Verification
- `uloop compile` → Success true, ErrorCount 0
- `uloop run-tests --filter-type regex --filter-value PausePointCompiledLineMapWarningTests` → TestCount 3, PassedCount 3, FailedCount 0
- `uloop run-tests --filter-type regex --filter-value` HotReload EditMode namespace → TestCount 344, PassedCount 344, FailedCount 0
- `uloop run-tests --filter-type regex --filter-value` Editor EditMode namespace → TestCount 2806, PassedCount 2798, FailedCount 0, SkippedCount 8 (suite Status Skipped / Success false because of those 8 skips; no failures)

<a href="https://cubic.dev/pr/hatayama/unity-cli-loop/pull/2213?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: Packages/src/Editor/FirstPartyTools/PausePoint/PausePointTools.cs [hunk 1]
```diff
@@ -521,10 +521,16 @@ private static PausePointResponse EnableBySourceLocation(EnablePausePointSchema
...
             if (!resolveResult.Success)
             {
-                return CreateValidationFailure(
+                PausePointResponse response = CreateValidationFailure(
                     resolveResult.ErrorMessage,
                     SourcePausePointConstants.ErrorCodeResolveFailed,
                     SourcePausePointConstants.ResolveFailedRecommendedNextAction);
+                if (shimLookup != null)
+                {
+                    response.Warning = BuildCompiledLineMapWarningOrEmpty(true, parameters.File);
```
[the remark this answers is not in the document]
Comment (hatayama) [reply Packages/src/Editor/FirstPartyTools/PausePoint/PausePointTools.cs]: Agreed. Both call sites now pass the real condition into the helper so the empty-warning branch is reachable in production. Resolve-failure uses `shimLookup != null`; the enable-success path uses `hasActiveHotReloadPatches && !retargetedToHotReloadPatch`. Behavior is unchanged.
